src/chiltepin/jedi/qg/wrapper.py

```diff
@@ -0,0 +1,197 @@
+            perl -p -i -e 's/(.* PROJECT ioda)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT crtm)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT ufo)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT gsw)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT fv3)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT femps)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT soca)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT mom6)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.* PROJECT coupling)/#\\1/g' ../src/CMakeLists.txt
+            perl -p -i -e 's/(.*mpas)/#\\1/ig' ../src/CMakeLists.txt
+            ecbuild -DCMAKE_INSTALL_PREFIX=../ ../src
+            echo Completed at $(date)
+            """
+            )
+
+        return bash_app(configure, executors=executors)
+
+    def get_make_task(
+        self,
+        executors=["serial"],
+    ):
+        def make(
+            stdout=None,
+            stderr=None,
+            jobs=8,
+            configure=None,
+            parsl_resource_specification={"num_nodes": 1},
+        ):
+            return self.environment + textwrap.dedent(
+                f"""
+            echo Started at $(date)
+            echo Executing on $(hostname)
+            spack env list
+            spack env deactivate
+            cd {self.install_path}/jedi-bundle/{self.tag}/build
+            make -j{jobs} VERBOSE=1
+            make install
+            echo Completed at $(date)
+            """
+            )
+
+        return bash_app(make, executors=executors)
+
+    def get_install_task(
+        self,
+        clone_executors=["service"],
+        configure_executors=["service"],
+        make_executors=["serial"],
+    ):
+        def install(
+            jobs=8,
+            stdout=None,
+            stderr=None,
+        ):
+            clone_task = self.get_clone_task(executors=clone_executors)
+            configure_task = self.get_configure_task(executors=configure_executors)
+            make_task = self.get_make_task(executors=make_executors)
+
+            clone = clone_task(
+                stdout=(stdout, "w"),
+                stderr=(stderr, "w"),
+            )
+            configure = configure_task(
+                stdout=(stdout, "a"),
+                stderr=(stderr, "a"),
+                clone=clone,
+            )
+            make = make_task(
+                jobs=jobs,
+                stdout=(stdout, "a"),
+                stderr=(stderr, "a"),
+                configure=configure,
+            )
+            return make
+
+        return join_app(install)
+
+    def clone(
+        self,
+        stdout=None,
+        stderr=None,
+        executors=["service"],
+    ):
+        return self.get_clone_task(executors)(
+            stdout=stdout,
+            stderr=stderr,
+        )
+
```

Comment:
```suggestion
        return self.get_clone_task(executors=executors)(
            stdout=stdout,
            stderr=stderr,
        )

```

How does this work when passing `stdout` and `stdderr`? Does `self.get_clone_task(executors)` return a function call to pass these parameters to?